.travis.yml

```diff
@@ -7,29 +7,23 @@ git:
 matrix:
   include:
     - os: linux
-      dist: xenial
+      dist: ubuntu
       sudo: required
       compiler: gcc
-      env: COVERAGE=1 CMAKE_OPTIONS="-DCMAKE_BUILD_TYPE=Debug -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=ON -DEXIV2_ENABLE_CURL=ON -DCMAKE_CXX_STANDARD=98"
+      env: COVERAGE=0 CMAKE_OPTIONS=".. -DCMAKE_BUILD_TYPE=Debug -DEXIV2_ENABLE_VIDEO=ON -DEXIV2_ENABLE_WEBREADY=ON -DEXIV2_BUILD_UNIT_TESTS=ON -DBUILD_WITH_COVERAGE=OFF -DEXIV2_ENABLE_CURL=ON -DCMAKE_CXX_FLAGS=-Wno-deprecated"
```

Comment:
Hi Robin. The `.. ` needs to be removed from this variable. You can see in the file `ci/run.sh` that we are already indicating the path relatively there:

```bash
# ci/run.sh line 25
cmake ${CMAKE_OPTIONS} -DEXIV2_TEAM_WARNINGS_AS_ERRORS=ON -DCMAKE_INSTALL_PREFIX=install ..
```

Reply:
I thought I had dealt with this.  Crap.  I'll fix it.

Do you know how to get the CI to build MinGW/msys2 and/or Cygwin64?

Reply:
The CI is mysteriously crashing on iotest in the new python version of bash_tests.  I haven't reproduced this on my machines at home.

I'll work on that tomorrow and then the CI will work.  I want to get back to the book this week as there is still a lot to be done.  We're having 5 days vacation starting Tuesday 22 September.